spec/cases/each_with_threads_and_exceptions.rb

```diff
@@ -0,0 +1,12 @@
+require './spec/cases/helper'
```

Comment:
pretty much copy paste, maybe make these pairs into 1 file and pass in `ENV['THREAD']`